frameworks/PHP/symfony/src/Controller/BenchController.php

```diff
@@ -2,108 +2,134 @@
 
 namespace App\Controller;
 
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait;
+use App\Entity\Fortune;
+use App\Entity\World;
+use Exception;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use App\Entity\Fortune;
-use App\Entity\World;
+use Symfony\Component\Routing\Annotation\Route;
 
-class BenchController extends Controller
+class BenchController extends AbstractController
 {
-    use ControllerTrait;
-
-    public function plaintextAction()
+    /**
+     * @Route("/plaintext")
+     */
+    public function plaintextAction(): Response
```

Comment:
best practice is to not use the `Action` suffix anymore

Reply:
thanks, i've missed that, will fix

Reply:
missed this, will fix